packages/ui/src/stories/Button.stories.tsx

```diff
@@ -198,6 +199,107 @@ export const CustomSizes: ComponentStory<typeof Button> = () => (
   </GridLayout>
 )
 
+export const IconButton: ComponentStory<typeof Button> = () => (
+  <GridLayout style={{ background: Colors.gray30, padding: 50 }}>
+    <GridLayout.Column span={4}>
+      <StackLayout gap={1}>
+        <Text type="heading 3">Primary Button</Text>
+        <Wrapper>
+          <Button
+            appearance="primary"
+            variant="large"
+            size="large"
+            icon="settings"
+          />
+
+          <Button
+            appearance="primary"
+            variant="medium"
+            size="medium"
+            icon="settings"
+          />
+        </Wrapper>
+      </StackLayout>
+    </GridLayout.Column>
+    <GridLayout.Column span={4}>
+      <StackLayout gap={1}>
+        <Text type="heading 3">Primary Ghost (Transparent)</Text>
+        <Wrapper>
```

Comment:
I need a row here with a gap, so I used GridLayout.Column with margin. ColumnLayout makes button full width

Reply:
or div instead of BoxLayout

Reply:
@aleksaToljic  Still not getting the look that I wanted because ColumnLayout sets  flex-grow: 1 to children and can't achieve auto width

![image](https://user-images.githubusercontent.com/42462482/175898691-8d5a0034-c882-4446-bd1c-daf4619a9c05.png)

Maybe to add "width" prop to ColumnLayout that will be full width by default, and can also be auto width?